src/main.rs

```diff
@@ -55,6 +55,10 @@ struct Cli {
 	/// Disable searching through Home Manager packages
 	#[clap(long = "no-home-manager-packages", action = ArgAction::SetFalse)]
 	home_manager_packages: bool,
+
+	/// Filter out pull requests that are not updates
+	#[clap(long = "updates-only")]
```

Comment:
```suggestion
	#[clap(long = "updates-only")]
```

Why does the long flag name need to be specified here? Iirc that is what it will default to in this case. Also, would a short version of the flag not be useful?

Reply:
Not sure why it needs to be specified honestly, just followed the previous code style. also short flag would be `-u` which is already in use, do you have any preferences as to what it should be?

Reply:
The previous code style in question?
![image](https://github.com/uncenter/nixpkgs-using/assets/47499684/cf4b3b77-2737-4a3b-97ce-ab0b8f89a86e)

If you are referring to https://github.com/uncenter/nixpkgs-using/blob/349f6bf31cc3ab468c014ef6f7e09e2053f0f68f/src/main.rs#L50-L57, those are specifically different and have a comment linking to an article on why. If you look closely you'll see that the reason for the specified long version is that the normal and `--no` versions are switched.

Reply:
You are right though I forgot there already was a `-u` in that case we can skip the short version :)

Reply:
ahhh i see